install.sh

```diff
@@ -215,6 +215,34 @@ semverParse() {
 	patch="${patch%%[-.]*}"
 }
 
+
+# on RHEL machines, the $releasever yum/dnf variable may have a Server or Client
+# suffix. Our .repo files use $releasever to switch between repositories for
+# CentOS/RHEL 7 and 8, but (currently) don't have separate repositories for
+# Client/Server variants.
+#
+# adjust_repo_releasever substitutes ${releasever} with 7 or 8
+adjust_repo_releasever() {
+	case $1 in
+	7*)
+		releasever=7
+		;;
+	8*)
+		releasever=8
+		;;
+	*)
+		# fedora, or unsupported
+		return
+		;;
+	esac
+
+	for channel in "stable" "test" "nightly"; do
+		yum-config-manager --setopt="docker-ce-${channel}.baseurl=${DOWNLOAD_URL}/linux/centos/${releasever}/\$basearch/${channel}" --save;
```

Comment:
We should prepend `sudo ` to these calls.

Reply:
Ah, good catch; let me add the `$sh_c` before, which looks like it automatically does the su / sudo